src/frontend/next/src/components/SnackBar/SnackBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, useState } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+type SnackbarProps = {
+  message: string;
+};
+
+const SimpleSnackbar: FC<SnackbarProps> = (props) => {
```

Comment:
We can drop line 11 and do proper typing here I think:

```ts
const SimpleSnackbar: FC<SnackbarProps> = ({ message } : SnackbarProps) => {
```

Reply:
@humphd : I agree we don't need another line. But I think just this is enough. TS will recognize the props passed to FC
```
const SimpleSnackbar: FC<SnackbarProps> = ({ message }) => {

```

Reply:
@humphd : I have resubmitted.